aws/registry.go

```diff
@@ -85,9 +85,11 @@ func (r *Registry) load() (*registryContents, error) {
 		// already existing on all interfaces and timestamped
 		// at the golang epoch
 		free, err := FindFreeIPsAtIndex(0, false)
-		if err == nil {
+		if err == nil || len(free) > 0 {
```

Comment:
why isn't this && ?

Reply:
I updated `FindFreeIPsAtIndex` to return partial results of ips found up until the error, so this lets us track anything we had found before we got an error